@here/harp-text-canvas/lib/rendering/TextStyle.ts

```diff
@@ -55,13 +55,38 @@ export enum VerticalAlignment {
 
 /**
  * Horizontal alignment to be used when placing text.
+ *
+ * @note Horizontal alignment value is exactly opposite to [[HorizontalPlacement]] value,
+ * cause when you place text on the right side of point (or icon) you normally want to have it
+ * left-aligned.
+ * @note Alignment attributes `hAlignment` and `vAlignment` are now depreciated in Theme
```

Comment:
Remove this note.
This module has no connection to Themes and technique.

Reply:
... After reading whole patch i think whole change to TextStyle can be removed as you've already resolve "placement" to "alignment" in higher layers.